Packs/Campaign/Scripts/FindEmailCampaign/FindEmailCampaign.yml

```diff
@@ -150,6 +150,12 @@ outputs:
 - contextPath: EmailCampaign.incidents.emailfromdomain
   description: The domains of the email senders involved in the campaign.
   type: Unknown
+- contextPath: EmailCampaign.incidents.recipients
+  description: The recipients of the emails involved in the campaign.
```

Comment:
A list of email addresses of recipients involved in the campaign.  The list is comprised of the following fields, "Email To", "Email CC", "Email BCC".